Products/CMFPlone/CatalogTool.py

```diff
@@ -285,6 +285,16 @@ def getIcon(obj):
 
 
 @indexer(Interface)
+def MimeType(obj):
+    if obj.portal_type in ['File','Image']:
```

Comment:
Why limit it to File and Image? Custom types could have a file or image field too. I would check to see if the object has a primary field, and if so use the mimetype from that field's value.